### PR TITLE
Reformat `insert_area.sql` into a function with py wrapper and tests

### PR DESCRIPTION
**Pull request by zlochina**
- [x] create a PostgreSQL function/procedure, including the header comment block (see `function_select_network_nodes_in_area.sql`
- [x] create a wrapper script
- [x] remove insert_area.sql
- [x] test
- [x] update the reference documentation in `reference.md`

Also resolved issue 45 by removing testing .sql file of `function_select_network_nodes`

### CODE REVIEW

File: SQL/functions/function_insert_area.sql [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  PERFORM insert_area($1, $2, NULL, $3);
+END;
+$$ LANGUAGE plpgsql;
+
+CREATE OR REPLACE FUNCTION insert_area(varchar, json) RETURNS VOID AS
+$$
+BEGIN
+  PERFORM insert_area(NULL, $1, NULL, $2);
+END;
+$$ LANGUAGE plpgsql;
```
Comment (zlochina) [inline SQL/functions/function_insert_area.sql:41]: Is overloading okay here?

File: python/roadgraphtool/insert_area.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        help="The description of the area",
+    )
+    return parser.parse_args()
+
+
+def main():
+    args = parse_arguments()
+
+    # Read the GeoJSON file
+    geojson = read_json_file(args.file_path)
```
Comment (zlochina) [inline python/roadgraphtool/insert_area.py:80]: Was also thinking about adding some geojson validator function. What's your opinion?

File: python/roadgraphtool/insert_area.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    parser.add_argument(
+        "-i", "--id", type=int, required=False, default=None, help="The id of the area"
+    )
+    parser.add_argument(
+        "-d",
+        "--description",
+        required=False,
+        default=None,
+        help="The description of the area",
+    )
```
Comment (zlochina) [inline python/roadgraphtool/insert_area.py:72]: Is execution with the help of flags suit us? Something like:
```sh
python insert_are.py -n "Area Name" -f path/to/geojson/file.json -i 1 -d "Area description"
```
or
```sh
python insert_area.py --name "Area Name" --file path/to/geojson/file.json --id 1 --description "Area description"
```

File: SQL/functions/function_insert_area.sql (hunk 1 above)
Comment (F-I-D-O) [reply SQL/functions/function_insert_area.sql:41]: It is possible, but I think optional parameters would fit better here

File: python/roadgraphtool/insert_area.py (hunk 3 above)
Comment (F-I-D-O) [reply python/roadgraphtool/insert_area.py:72]: I think it is a good solution

File: python/roadgraphtool/insert_area.py (hunk 2 above)
Comment (F-I-D-O) [reply python/roadgraphtool/insert_area.py:80]: nice thing to have in the future, but right now, I think it is not necessary

File: python/roadgraphtool/insert_area.py [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+def main():
+    args = parse_arguments()
+
+    # Read the GeoJSON file
+    geojson = read_json_file(args.file_path)
+
+    # Establish database connection
+    conn = (
+        get_database_connection()
+    )  # TODO import connection flow from ../scripts/main.py
```
[the remark this answers is not in the document]
Comment (F-I-D-O) [reply python/roadgraphtool/insert_area.py:85]: I think you should use the functions from `db.py` for all database operations

File: SQL/functions/function_insert_area.sql (hunk 1 above)
Comment (zlochina) [reply SQL/functions/function_insert_area.sql:41]: Updated

File: python/roadgraphtool/insert_area.py (hunk 2 above)
Comment (zlochina) [reply python/roadgraphtool/insert_area.py:80]: Should I leave this suggestion somewhere to not forget?

**Comment by zlochina** [comment]
1 out of 4 tests is failed due to [issue](https://github.com/aicenter/road-graph-tool/issues/46)

**Comment by zlochina** [comment]
It is ready to be merged. Py wrapper was fixed in the last commit to perform as expected

File: python/roadgraphtool/insert_area.py [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+                description=args.description,
+                geom=geojson,
+            )
+        conn.commit()
+        print(f"Area '{args.name}' inserted successfully.")
+    except psycopg2.Error as e:
+        conn.rollback()
+        print(f"Error inserting area: {e}")
+    finally:
+        conn.close()
```
[the remark this answers is not in the document]
Comment (zlochina) [reply python/roadgraphtool/insert_area.py:102]: Particularly, this issue is raised `Error inserting area: Query missing geometry column 'geom'`

**Pull request merged**